### PR TITLE
Add example using string expressions

### LINKED ISSUE #135: Interacting with data: String expressions
A runnable example show-casing how one can use string expressions.

Path: /examples/data/string-expressions

### PR DESCRIPTION
**Pull request by wennmo**
Fixes #135.

### Status
```
[ ] Under development
[x] Waiting for code review
[ ] Waiting for merge
```
### Information
```
[ ] Contains breaking changes
[ ] Contains new API(s)
[x] Contains documentation
[ ] Contains test
```

### CODE REVIEW

File: examples/data/string-expression/string-expression.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const createSession = require("../../session");
+
+const qlikScript = `
```
Comment (marcusoffesson) [inline examples/data/string-expression/string-expression.js]: Naming suggestion: `loadScript`.

File: examples/data/string-expression/string-expression.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+TempTable:
+Load
+RecNo() as ID
+AutoGenerate 100
+`;
+
+const stringExpressionProperties = {
```
Comment (marcusoffesson) [inline examples/data/string-expression/string-expression.js]: How about `objectProperties` or just `properties`?

File: examples/data/string-expression/string-expression.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+session
+  .open()
+  .then(global => global.createSessionApp())
+  .then(doc => {
+    // Load in some data into the session document:
+    doc.setScript(qlikScript).then(() => doc.doReload())
+      // We create a string expression using
+      // a field in the data we loaded:
+      .then(() => doc.createObject(stringExpressionProperties))
+      .then(stringExpression => {
```
Comment (marcusoffesson) [inline examples/data/string-expression/string-expression.js]: Ambiguous naming IMHO, how about simply calling it `object`?

File: examples/data/string-expression/string-expression.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        // Just a helper function that helps fetching the layout and prints out
+        // the evaluated string expression.
+        const update = () =>
```
Comment (marcusoffesson) [inline examples/data/string-expression/string-expression.js]: Not sure I understand the need for this helper. Wouldn't

```js
doc.setScript(script).then(() => doc.doReload())
  .then(() => doc.createObject(properties))
  .then(object => object.getLayout())
  .then(layout => console.log(...))
  .then(() => session.close())
```
work just as well?

File: examples/data/string-expression/string-expression.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
+const createSession = require('../../session');
+
+const qlikScript = `
```
Comment (marcusoffesson) [inline examples/data/string-expression/string-expression.js]: Naming suggestion: `loadScript` or just `script`.

File: examples/data/string-expression/string-expression.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  .then(global => global.createSessionApp())
+  .then((doc) => {
+    // Load in some data into the session document:
+    doc
+      .setScript(qlikScript)
+      .then(() => doc.doReload())
+      // We create a string expression using
+      // a field in the data we loaded:
+      .then(() => doc.createObject(stringExpressionProperties))
+      .then((stringExpression) => {
```
Comment (marcusoffesson) [inline examples/data/string-expression/string-expression.js]: Ambiguous naming IMHO, how about simply calling it `object`?

File: examples/data/string-expression/string-expression.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+        // Just a helper function that helps fetching the layout and prints out
+        // the evaluated string expression.
+        const update = () =>
```
Comment (marcusoffesson) [inline examples/data/string-expression/string-expression.js]: Not sure I understand the need for this helper. Wouldn't

```js
doc.setScript(script).then(() => doc.doReload())
  .then(() => doc.createObject(properties))
  .then(object => object.getLayout())
  .then(layout => console.log(...))
  .then(() => session.close())
```
work just as well?

File: examples/data/string-expression/string-expression.js (hunk 5 above)
Comment (peol) [inline examples/data/string-expression/string-expression.js]: It's taken from other examples, let's rename this in all examples in a separate PR instead

File: examples/data/string-expression/string-expression.js (hunk 7 above)
Comment (peol) [inline examples/data/string-expression/string-expression.js]: Taken from list-object example which outputs the layout multiple times, I believe this example doesn't need that

**Pull request merged**